addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -227,6 +233,9 @@ const threadPatch = {
         }
         return super.avatarUrl;
     },
+    get showCorrespondentCountry() {
```

Comment:
The whole `showCorrespondentCountry` stuff should be moved as overrides to livechat, we don't use that at all in mail.